feat(media): add `neocord.nvim` (#1049)
AstroNvim/astrocommunity: lua/astrocommunity/media/neocord-nvim/README.md

# neocord.nvim

> 🛰️ Discord Rich Presence plugin for Neovim written in Lua

Reviewer: Should have a required dependency list

> 🛰️ Discord Rich Presence plugin for Neovim written in Lua

Reviewer: Discord Rich Presence for Neovim (Fork of presence.nvim)

```suggestion
Discord Rich Presence for Neovim (Fork of presence.nvim)
```
